test/cypress/integration/simplified-account/service-settings/card-payments/updates.cy.js

```diff
@@ -0,0 +1,103 @@
+const {
+  setupStubs,
+  USER_EXTERNAL_ID,
+  GATEWAY_ACCOUNT_ID,
+  SERVICE_EXTERNAL_ID,
+  ACCOUNT_TYPE
+} = require('@test/cypress/integration/simplified-account/service-settings/card-payments/util')
+const {
+  patchUpdateDefaultBillingAddressCountrySuccess,
+  patchUpdateCollectBillingAddressSuccess
+} = require('@test/cypress/stubs/service-stubs')
+const { patchAccountUpdateApplePaySuccess, patchAccountUpdateGooglePaySuccess } = require('@test/cypress/stubs/gateway-account-stubs')
+
+const baseUrl = `/simplified/service/${SERVICE_EXTERNAL_ID}/account/${ACCOUNT_TYPE}/settings/card-payments`
+
+describe('Card payment updates', () => {
+  beforeEach(() => {
+    cy.task('clearStubs')
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+  })
+
+  it('should allow update of Collect billing address', () => {
```

Comment:
I think these tests should be expanded upon. Each individual setting should have a `describe('update <setting> page')`, with tests for an admin and non-admin user (the non-admin user tests should just test that the app 403s). There should be a few basic tests in the admin user test that the heading and title are correct, and test any validation on the forms as well as the happy path.